doc/get-started.md

```diff
@@ -115,21 +115,42 @@ For more information on installing Reader SDK for Android, see the
       ...
     }
     ```
-1. Extend the Android Application class (`android.app.Application`) and add code
-   to Import and initialize Reader SDK:
+1. Open or create `MainApplication` class in your project, make sure extend the Flutter Application class (`FlutterApplication`) and add code
```

Comment:
Suggest smoothing this sentence out a bit:
```
Open the `MainApplication` class for your project. **Note that `MainApplication`
is not created by default through `flutter create <Project>`, so you may need to
create it**. Add code to extend the Flutter Application class
(`FlutterApplication`) that imports and initializes Reader SDK:
```

Reply:
updated!